Classes/Issues/IssueManagingContextController.swift

```diff
@@ -363,10 +399,24 @@ final class IssueManagingContextController: NSObject, ContextMenuDelegate {
         } else if let people = viewController as? PeopleViewController {
             didDismiss(controller: people)
         } else if let labels = viewController as? LabelsViewController {
-            didDismiss(selected: labels.selected)
+            didDismiss(controller: labels)
         }
     }
 
     func contextMenuDidDismiss(viewController: UIViewController, animated: Bool) {}
 
+    func showCancelAlert(onKeep: @escaping () -> Void) {
```

Comment:
What about `completion` rather than `onKeep`, which provides a `Bool` result? Reading this I'm wondering if "on keep" is easy to understand.

Reply:
Agreed!